### ISSUE TITLE
Google Maps causing system crashes in A12

### ISSUE THREAD

**Issue by Hi1307** [phhusson/treble_experimentations#2121]
Google Maps is very slow in all A12 GSIs, causing phone to freeze and restart when scrolling around in the app. 

Not sure if it's a qcom GPU blob issue, will attach logcat when I can get the phone to not reboot.

Device: Mi 11 Lite 5G (renoir)
GPU: qcom Adreno 642
Version: 12 400.d arm64 ab (with gapps), same issue in 400a and 400c
Google Maps version: 11.11.2 (Google Play)

**Comment by ponces** [comment]
Do you have the same issue on v400.e?

**Comment by Hi1307** [comment]
Yes (sorry for late reply, was waiting for the Pixel Experience build for
v400.e)


CONFIDENTIALITY: If this email has been sent to you by mistake, please
notify the sender and delete it immediately. As it may contain confidential
information, the retention or dissemination of its contents may be an
offence under the Official Secrets Act.

**Comment by Hi1307** [comment]
[logcat-googlemaps.txt](https://github.com/phhusson/treble_experimentations/files/7799063/logcat-googlemaps.txt)

Here's the logcat, the crash seems to be a memory issue?
